core/packages/test/scripts/set-env.sh

```diff
@@ -1,10 +1,12 @@
+#!/usr/bin/env sh
+
+# exports enable use in core/packages/test/config/launch-config.toml
 root_dir="$(realpath ../../..)"
-relaychain_dir="${POLKADOT_DIR:-$root_dir/relaychain}"
-relaychain_version="${POLKADOT_VER:-v0.9.30}"
-relaychain_bin="${POLKADOT_BIN:-$relaychain_dir/target/release/polkadot}"
 parachain_dir="$root_dir/parachain"
 parachain_runtime="${PARACHAIN_RUNTIME:-snowbase}"
 parachain_bin="$parachain_dir/target/release/snowbridge"
+relaychain_version="${POLKADOT_VER:-v0.9.30}"
+relaychain_bin="${POLKADOT_BIN:-$parachain_dir/.cargo/bin/polkadot.d/$relaychain_version/polkadot}"
```

Comment:
Including `$relaychain_version` here allows us to swap out polkadot versions by changing `POLKADOT_VER`. The way `relaychain_bin` gets checked, we won't rebuild the same polkadot version.